### PR TITLE
REL pyfolio 0.8.0

### LINKED ISSUE #426: Add pyfolio/perf_attrib.py to __init__.py

### PR DESCRIPTION
**Pull request by eigenfoo**
@twiecki @vikram-narayan I also updated `__init__.py` while I was at it.

Closes #426

### CODE REVIEW

**Comment by twiecki** [comment]
Great, want to wait for https://github.com/quantopian/pyfolio/pull/425 but think we should go ahead then. Also requesting engineering review @richafrank.

**Comment by yankees714** [comment]
@eigenfoo Noticed an issue while attempting to install from head of master, we're missing a comma after `'scikit-learn>=0.18.2'` in the `install_reqs`: https://github.com/quantopian/pyfolio/blob/0ea262f93251f015d70c694d0bd18654bddd7df2/setup.py#L47-L58

File: WHATSNEW.md [hunk 1]
```diff
@@ -2,6 +2,38 @@
 
 These are new features and improvements of note in each release.
 
+## v0.8.0 (Aug 23rd, 2017)
+
+This is a major release from `0.7.0`, and all users are recoommended to upgrade.
```
Comment (yankees714) [inline WHATSNEW.md]: Typo "recoommended"

Comment (eigenfoo) [inline WHATSNEW.md]: Oops, fixed.

**Comment by eigenfoo** [comment]
@yankees714 @twiecki fixed the bugs

**Comment by eigenfoo** [comment]
@twiecki?

**Comment by twiecki** [comment]
Waiting to merge https://github.com/quantopian/pyfolio/pull/425.

**Comment by eigenfoo** [comment]
There was a small kerfuffle with #434: reverted the commit where I made the input checker.

**Comment by twiecki** [comment]
Woohoo!

**Comment by eigenfoo** [comment]
@twiecki https://github.com/quantopian/pyfolio/releases shows we're still on `0.7.0`

**Comment by twiecki** [comment]
OK, should be on github and pypi: https://github.com/quantopian/pyfolio/releases/tag/v0.8.0. Thanks for the help @eigenfoo and @jameschristopher.

**Pull request merged**